peachjam/download.py

```diff
@@ -57,9 +57,10 @@ def download_file_from_google(file_id):
 def download_with_urllib(url):
     try:
         f = NamedTemporaryFile()
-        urllib.request.urlretrieve(url, f.name)
-    except urllib.error.HTTPError as e:
+        r = requests.get(url)
+        r.raise_for_status()
+        f.write(r.content)
+        return f
+    except requests.exceptions.HTTPError as e:
         logger.error(e)

```

Comment:
Surely downloading the file is a key part of the import? If that fails, the import shouldn't succeed, I would think?

Reply:
@longhotsummer I have been grappling with what to do here. The issue is that the whole import fails if only one file is missing and the error is not shown to the user. Should we fail the whole import if one file is missing?